reader.go

```diff
@@ -138,11 +138,19 @@ func (r *Reader) Read() (File, error) {
 		lineLength := len(line)
 
 		switch {
-		case r.lineNum == 1 && lineLength > RecordLength && lineLength%RecordLength == 0:
-			if err := r.processFixedWidthFile(&line); err != nil {
+		case r.lineNum == 1 && lineLength > RecordLength:
+			extraChars := lineLength % RecordLength
+			if extraChars != 0 {
+				err := fmt.Errorf(
```

Comment:
@adamdecaf I've replaced `RecordWrongLengthErr` with this error since the former will print out an incorrect message for this case, it would output something like `must be 94 characters and found 471`.

This change outputs `1 extra character(s) in ACH file: must be 470 but found 471`

Reply:
Cool!